src/shapes/star.js

```diff
@@ -0,0 +1,15 @@
+// Star helper function from https://p5js.org/examples/form-star.html
+module.exports = function drawStar(p5, x, y, radius1, radius2, numPoints) {
+  let angle = p5.TWO_PI / numPoints;
```

Comment:
Do any of the other shapes use p5? Thematically, it makes sense to put this here, but I believe it breaks an established pattern.

Reply:
I guess it could also just be extracted as a helper function within Effects.js, like `lerpColorFromPalette()`. Do you think that would be better?

Reply:
I like your idea to group them thematically.